mgmt_cli_test.py

```diff
@@ -537,3 +657,110 @@ def test_enospc_during_backup(self):
+                                        'Error: {}'.format(result))
+            if directory_size_result.stdout:
+                directory_size = directory_size_result.stdout[:directory_size_result.stdout.find("\t")]
+                self.log.debug(f"Removed the directory of keyspace {keyspace_name} from node "
+                               f"{db_node}\nThe size of the directory is {directory_size}")
+
+        finally:
+            db_node.start_scylla_server(verify_up=True, verify_down=False)
+
+    def _insert_data_while_excluding_each_node(self, total_num_of_rows, keyspace_name="keyspace2"):
+        """
+        The function split the number of rows to the number of nodes (minus 1) and in loop does the following:
+        shuts down one node, insert one part of the rows and starts the node again.
+        As a result, each node that was shut down will have missing rows and will require repair.
+        """
+        num_of_nodes = self.params.get("n_db_nodes")
+        num_of_rows_per_insertion = int(total_num_of_rows / (num_of_nodes - 1))
+        stress_command_template = "cassandra-stress write cl=QUORUM n={} -schema 'keyspace={} replication(factor=3)'" \
+                                  " -col 'size=FIXED(1024) n=FIXED(1)' -pop seq={}..{} -port jmx=6868 -mode cql3" \
+                                  " native -rate threads=200 -log interval=5"
+        start_of_range = 1
+        # We can't shut down node 1 since it's the default contact point of the stress command, and we have no way
+        # of changing that. As such, we skip it.
+        for node in self.db_cluster.nodes[1:]:
+            self.log.info(f"inserting {num_of_rows_per_insertion} rows to every node except {node.name}")
+            end_of_range = start_of_range + num_of_rows_per_insertion - 1
+            node.stop_scylla_server(verify_up=False, verify_down=True)
+            stress_thread = self.run_stress_thread(stress_cmd=stress_command_template.format(num_of_rows_per_insertion,
+                                                                                             keyspace_name,
+                                                                                             start_of_range,
+                                                                                             end_of_range))
+            time.sleep(15)
+            self.log.info(f'load={stress_thread.get_results()}')
+            node.start_scylla_server(verify_up=True, verify_down=False)
+            start_of_range = end_of_range + 1
+        with self.db_cluster.cql_connection_patient(self.db_cluster.nodes[0]) as session:
+            session.execute(f"ALTER TABLE {keyspace_name}.standard1 WITH read_repair_chance = 0.0")
+
+        for node in self.db_cluster.nodes:
+            node.run_nodetool("flush")
+
+    def create_fault(self, delete_from_multiple_nodes, keyspace_to_be_repaired=None):
```

Comment:
fault is a bit general, it's more like `unrepair_cluster` or something more like that. fault sound like you are running nemesis or something similar.

Reply:
`create_missing_rows_in_cluster` is much much better